src/Compilers/CSharp/Test/Semantic/FlowAnalysis/RegionAnalysisTests.cs

```diff
@@ -145,6 +145,144 @@ static void Main(string[] args)
             Assert.Null(GetSymbolNamesJoined(dataFlowAnalysisResults.CapturedOutside));
         }
 
+        [Fact]
+        public void DataFlowsInAndNullable_Property_InConstructor()
```

Comment:
>DataFlowsInAndNullable_Property_InConstructor [](http://example.com/codeflow?start=20&length=45)

Please confirm that there is np behavior change in for the new test scenarios.

Reply:
@AlekseyTs I added them when in `main` branch and they were passing. They exercise some code paths in `AbstractFlowAnalysis`. I think I should also add prototypes around these tests to verify behavior of similar tests for semi-auto properties.